backend/api/Helpers/Reporting/ReportHelper.cs

```diff
@@ -34,11 +34,12 @@ public static ContentResult GenerateCsv<T>(IEnumerable<T> items)
         /// </summary>
         /// <typeparam name="T"></typeparam>
         /// <param name="items"></param>
+        /// <param name="sheetName"></param>
         /// <returns></returns>
-        public static FileStreamResult GenerateExcel<T>(IEnumerable<T> items)
+        public static FileStreamResult GenerateExcel<T>(IEnumerable<T> items, string sheetName)
         {
-            var data = items.ConvertToDataTable();
-            var excel = data.ConvertToXLWorkbook("Properties");
+            var data = items.ConvertToDataTable(sheetName);
+            var excel = data.ConvertToXLWorkbook(sheetName);
```

Comment:
We can now control the sheet name that Excel has for each export.

Reply:
Nice lil addition!